src/trunk.c

```diff
@@ -8816,68 +8834,101 @@ trunk_node_print_branches(trunk_handle *spl, uint64 addr, void *arg)
    trunk_node           node;
    trunk_node_get(spl->cc, addr, &node);
 
-   platform_log(
-      log_handle,
-      "------------------------------------------------------------------\n");
+   // clang-format off
+   const char *dashes = "------------------------------------------------------------------";
+   const char *dashes_long = "-------------------------------------------------------------------------------------------------";
```

Comment:
These are formatting diffs to account for longer field-widths for certain columns printed below.